bin/convertToIcns

```diff
@@ -48,23 +47,12 @@ trap cleanUp EXIT
 mkdir -p "${TEMP_DIR}"
 mkdir -p "${ICONSET}"
 
-PNG_PATH="${TEMP_DIR}/icon.png"
-${CONVERT_TO_PNG} "${SOURCE}" "${PNG_PATH}"
+sips -s format tiff "$SOURCE" --out "$TEMP_DIR/icon_128.tiff" -z 128 128 2>&1 > /dev/null & 
```

Comment:
Why the final `&`? I don’t see a point to it. If you remove that character you can remove the next three lines: the two comments and the `while`.